### PR TITLE
Fix travis yml

### PR DESCRIPTION
**Pull request by asajeffrey**
Gets the travis build to be green again (includes #152 and #153).

### CODE REVIEW

File: surfman/src/platform/windows/angle/surface.rs [hunk 1]
```diff
@@ -158,11 +159,11 @@ impl Device {
             ];
 
             EGL_FUNCTIONS.with(|egl| {
-                let egl_surface = if let Some(HandleOrTexture::Texture(texture)) = share_handle {
+                let egl_surface = if let Some(share_handle) = share_handle {
```
Comment (jdm) [inline surfman/src/platform/windows/angle/surface.rs:162]: These changes don't look like they compile when used with the opexnr webxr backend.

Comment (jdm) [reply surfman/src/platform/windows/angle/surface.rs:162]: https://github.com/jdm/surfman/commits/d3d-share is a branch that should address that.

**Comment by jdm** [comment]
If the goal is only to get the build working and the openxr changes will be in a separate PR, that's fine.

**Comment by asajeffrey** [comment]
This is just to get the build working again, atm everything is red on travis, which is suboptimal.

**Pull request merged**